resources/views/admin/book_titles/index.blade.php

```diff
@@ -0,0 +1,88 @@
+@extends('admin.layouts.master')
+
+@section('title', __('book_titles.title'))
+@section('page', __('book_titles.title'))
+
+@section('content')
+    <div class="row">
+        <div class="col-12">
+            @include('admin.shared.alert')
+            <form class="bg-gradient-white rounded row w-100 mx-0 my-3 py-3">
+                <div class="col-md-3">
+                    <input value="{{request('name')}}" type="text" class="form-control" name="name"
+                           placeholder="{{ __('book_titles.name') }}"/>
+                </div>
+                <div class="col-md-3">
+                    <input value="{{request('author')}}" type="text" class="form-control" name="author"
+                           placeholder="{{ __('book_titles.author') }}"/>
+                </div>
+                <div class="col-md-5">
+                    <select class="form-control" name="genre">
+                        <option value="*">{{ __('genres.all_genres') }}</option>
+                        @foreach($genres as $genre)
+                            @if(request('genre') === $genre)
+                                <option selected>{{$genre}}</option>
+                            @else
+                                <option>{{$genre}}</option>
+                            @endif
```

Comment:
```suggestion
                            <option {{ request('genre') === $genre ? 'selected' : '' }}>{{$genre}}</option>
```

Em có thể chuyển như sau cho ngắn nếu muốn.